questionnaire/repo.go

```diff
@@ -141,5 +147,11 @@ func (r *repo) saveAnswer(
 	userID uuid.UUID,
 	answer Answer) error {
 
-	return errors.New("not implemented")
+	db := r.db.Unsafe()
+	_, err := db.ExecContext(ctx, "UPDATE user_answers SET raw_answer=$1, question_state='answered' WHERE user_id=$2 AND questionnaire_id=$3", answer, userID, qnrID)
```

Comment:
Сделай установку значения через параметр
```suggestion
	_, err := db.ExecContext(ctx, "UPDATE user_answers SET raw_answer=$1, question_state=$4 WHERE user_id=$2 AND questionnaire_id=$3", answer, userID, qnrID, answerStateAnswered)
```